waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates/Skeleton.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE TupleSections #-}
+
+module Wasp.Cli.Command.CreateNewProject.StarterTemplates.Skeleton
+  ( readWaspProjectSkeletonFiles,
+  )
+where
+
+import Data.Text (Text)
+import StrongPath (File', Path, Rel, System, reldir, (</>))
+import qualified Wasp.Data as Data
+import Wasp.Project.Common (WaspProjectDir)
+import Wasp.Util.IO (listDirectoryDeep, readFileStrict)
+
+readWaspProjectSkeletonFiles :: IO [(Path System (Rel WaspProjectDir) File', Text)]
+readWaspProjectSkeletonFiles = do
+  skeletonFilesDir <- (</> [reldir|Cli/templates/skeleton|]) <$> Data.getAbsDataDirPath
```

Comment:
I had trouble parsing this line at first, can write in way that it reads like a path i.e. from left to right?